cordapp/README.md

```diff
@@ -50,32 +60,34 @@ do some manual work to set up a docker container (instructions below).
    ```
    apt-get -y update && apt-get install -y openjdk-8-jdk-headless && export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
    ```
-5. Change directory to your project and run the project as you would under Linux:
+5. Change directory to your project and run the project.
     ```
     cd <project-directory>
     ./gradlew workflows:test
     ```
 
-Alternatively, ubuntu 20.04 via WSL 2 ([Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install)
+Alternatively, Ubuntu 20.04 via WSL 2 ([Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install)
```

Comment:
There is missing bracket ')' after **Windows Subsystem for Linux 2**

Reply:
Corrected.